src/persistence/SubjectExecutor.ts

```diff
@@ -1117,7 +1123,7 @@ export class SubjectExecutor {
             // merge into entity all generated values returned by a database
             if (subject.generatedMap)
                 this.queryRunner.manager.merge(
-                    subject.metadata.target as any,
+                    subject.metadata.target as EntityTarget<ObjectLiteral>,
```

Comment:
Just remove the cast, it was not necessary in the first place.

We prefer less code than additional unnecessary code ;)